modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -334,7 +334,7 @@ private void IssuesListContextOptionPressed(int id)
             }
         }
 
-        private void IssuesListRmbSelected(int index, Vector2 atPosition)
+        private void IssuesListClicked(int index, Vector2 atPosition, int mouse_button_index)
```

Comment:
Is hard-coding correct here? As I didn't find any enum for MouseButton in mono.

Reply:
I think the enums are generated from the class reference, converted to something C# friendly.
I didn't test but I think here it would be `ButtonList.Right`, and it likely needs to be cast to `int` so:
```csharp
if (mouseButtonIndex != (int)ButtonList.Right) {
	return;
}
```

Reply:
Not working :(

Reply:
In master `ButtonList` is `MouseButton`, also in C# the style is opening the bracket in the next line. So it should be like:

```csharp
if (mouseButtonIndex != (int)MouseButton.Right)
{
	return;
}
```

However, in https://github.com/Vitika9/godot/blob/c37a7fce8a2464e2e5356ff0635320e3c41633af/modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs#L381 we have this line: `_issuesList.AllowRmbSelect = true;`

Shouldn't we just not set `allow_rmb_select` to true (default is false) and then we won't even get the signal emitted? The same would apply to all other cases so the early return seems unnecessary since I don't think any of them set `allow_rmb_select` to true.

Reply:
> In master `ButtonList` is `MouseButton`, also in C# the style is opening the bracket in the next line. So it should be like:
> 
> ```cs
> if (mouseButtonIndex != (int)MouseButton.Right)
> {
> 	return;
> }
> ```
Thanks, I will fix this

Reply:
Ah I see, I missed that. So after looking at the `IssuesListClicked` method again I realize it opens a context menu popup, so I think it makes sense for the mouse's right button to trigger it, therefore I think there shouldn't be an early return. Sorry for the misunderstanding.

So, because I think a context menu should be opened with the mouse's right button I think either we don't have an early return or the early return checks for `MouseButton.Left` instead (but I don't know if users will be happy with that change so I'd go for the first option which would keep it working as before and we could consider changing the behavior in a future PR).